### PR TITLE
YouTube video duration using Google API

### PR DESCRIPTION
**Pull request by aarthi-axim**
Obtain `video duration` of `YouTube URL` using `Google API`.

### CODE REVIEW

File: lib/google/youtube/utils.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            },
+        )
+    else:
+        youtube_video_id = re_match.group(1) if re_match else None
+
+    return youtube_video_id
+
+
+def build_youtube_api_url(youtube_video_id):
+    base_url = 'https://www.googleapis.com/youtube/v3/videos?'
```
Comment (jontsai) [inline lib/google/youtube/utils.py:36]: Define in `lib/google/youtube/constants.py` as 

```
GOOGLE_APIS_YOUTUBE_BASE_URL = 'https://www.googleapis.com/youtube/v3'
GOOGLE_APIS_YOUTUBE_VIDEOS_URL = f'{GOOGLE_APIS_YOUTUBE_BASE_URL}/videos'
```

File: lib/google/youtube/utils.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    query = urllib.parse.urlencode(
+        {
+            'id': youtube_video_id,
+            'part': 'contentDetails',
+            'key': settings.HTK_GOOGLE_SERVER_API_KEY[0],
+        }
+    )
+
+    api_url = base_url + query
```
Comment (jontsai) [inline lib/google/youtube/utils.py:45]:
```suggestion
    url = f'{GOOGLE_APIS_YOUTUBE_VIDEOS_URL}?{query}`
```

File: lib/google/youtube/utils.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    return api_url
```
Comment (jontsai) [inline lib/google/youtube/utils.py:46]:
```suggestion
    return url
```

**Review by jontsai — CHANGES_REQUESTED** [review]

File: lib/google/youtube/utils.py (hunk 1 above)
Comment (aarthi-axim) [reply lib/google/youtube/utils.py]: Done

File: lib/google/youtube/utils.py (hunk 2 above)
Comment (aarthi-axim) [reply lib/google/youtube/utils.py]: Done

File: lib/google/youtube/utils.py (hunk 3 above)
Comment (aarthi-axim) [reply lib/google/youtube/utils.py]: Done

**Review by jontsai — APPROVED** [review]
LGTM

**Pull request merged**